controllers/managedocs_controller.go

```diff
@@ -70,6 +71,8 @@ const (
 	alertmanagerConfigName                 = "managed-ocs-alertmanager-config"
 	dmsRuleName                            = "dms-monitor-rule"
 	storageClassSizeKey                    = "size"
```

Comment:
This const name doesn't longer describe the usage as we now use it for two different use cases.
Converged -> storage class size
Consumer -> requested capacity size

Can we change the name to something more general? maybe `storageSizeKey`?